# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxFcPlatformFontList.h
# Commit: d51c979e9930
# Full Hash: d51c979e9930e3f20d601149c0cab64d748b872c
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2019-04-29 21:53:38
# Regressor Bug: 1514869
# File Overlap Count: 1
# Description:
#   Bug 1514869 - patch 4 - Implement Linux/fontconfig backend for the shared font list. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D22940
# ==============================================================================

diff -r c154853c599a -r d51c979e9930 gfx/thebes/gfxFcPlatformFontList.h
--- a/gfx/thebes/gfxFcPlatformFontList.h	Sat Apr 27 15:38:26 2019 +0000
+++ b/gfx/thebes/gfxFcPlatformFontList.h	Sat Apr 27 15:38:28 2019 +0000
@@ -270,6 +270,7 @@
 
   // initialize font lists
   nsresult InitFontListForPlatform() override;
+  void InitSharedFontListForPlatform() override;
 
   void GetFontList(nsAtom* aLangGroup, const nsACString& aGenericFamily,
                    nsTArray<nsString>& aListOfFonts) override;
@@ -277,6 +278,10 @@
   void ReadSystemFontList(
       InfallibleTArray<mozilla::dom::SystemFontListEntry>* retValue);
 
+  gfxFontEntry* CreateFontEntry(
+      mozilla::fontlist::Face* aFace,
+      const mozilla::fontlist::Family* aFamily) override;
+
   gfxFontEntry* LookupLocalFont(const nsACString& aFontName,
                                 WeightRange aWeightForEntry,
                                 StretchRange aStretchForEntry,
